src/main/java/seedu/address/logic/commands/deletecommand/DeleteMentorCommand.java

```diff
@@ -15,8 +15,8 @@
  */
 public class DeleteMentorCommand extends DeleteCommand {
 
-    public static final String COMMAND_WORD = "delete";
-    public static final String MESSAGE_INVALID_MENTOR_DISPLAYED_INDEX = "The mentor ID provided is invalid";
+    public static final String MESSAGE_INVALID_MENTOR_DISPLAYED_INDEX = "The mentor ID provided is "
```

Comment:
Oh so is it changed to just delete(the parser will automatically parse out the Entity type)? Do correct me if I am wrong hahaha 😄

Reply:
yup, the command_word is just delete. So when you do delete P-1, it automatically knows that P is associated with participant so it deletes Participant with ID P-1